pull_request_template.md

```diff
@@ -0,0 +1,37 @@
+# Description
+
+<!--- Please include a summary of the changes and the related issue. -->
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
+
+## Type of change
+
+**Select Type(s) of change:**
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Documentation update
+
+# How Has This Been Tested?
```

Comment:
I don't think this is needed either because we expect development to be done locally and tests ran would be the local tests.

Reply:
Idea behind this was so that we know if we have a poor test setup somewhere. For example if someone's doing something really odd to test because it isn't clear how that module should be tested. Think its too much?